lib/aquilon/worker/processes.py

```diff
@@ -50,17 +50,18 @@
 
 DSDB_ENABLED = config.getboolean("dsdb", "enable")
 
-if DSDB_ENABLED:
-    # FIXME - this needs to be moved to depends.py after
-    # refactoring runtests.py and Config to allow override
-    # sys.path for python modules when running tests
-    # DSDB python client
+try:
     import ms.version
+except ImportError:
+    pass
+else:
+    ms.version.addpkg('setuptools', '0.6c11')
```

Comment:
why this one is added? :)

Reply:
I don't know, I copied and pasted it from you?

Reply:
i might have copied if from somewhere else. My bad. Actually we might release this fix ourselves next week and push to github. Will let you know then we can rebase this one :)

Reply:
Cool